addon/active-model-adapter.js

```diff
@@ -148,7 +148,7 @@ const ActiveModelAdapter = RESTAdapter.extend({
     @param  {Object} payload
     @return {Object | DS.AdapterError} response
   */
-  handleResponse: function(status, headers, payload) {
+  handleResponse: function(status, headers, payload, requestData) {
```

Comment:
I disagree, I think `unused: true` is a really annoying setting. We often copy/paste from documentation as developers, and the unused argument gets minified out anyway.